op-acceptor/nat.go

```diff
@@ -102,25 +111,18 @@ func (n *nat) Start(ctx context.Context) error {
 	// Run tests immediately on startup
 	err := n.runTests()
 	if err != nil {
-		// For other errors, return an ExitCoder with exit code 1
-		n.config.Log.Error("Error running tests", "error", err)
-		return cli.Exit(fmt.Sprintf("Error running tests: %v", err), 2)
+		if strings.Contains(err.Error(), "test failures:") {
```

Comment:
Probably better to define a specific type for a test error and then do a type query here than inspect the error string.

Reply:
Agreed. Will do this in a follow-up PR so we can minimise the open branches to resolve this weird issue in CI we're seeing right now. Thanks